algo/README.md

```diff
@@ -313,7 +313,7 @@ Trials could lead to `WARN` in case knowing the exact trial treatment is require
 | MUTATION_IN_GENE_X_IN_EXON_Y                                                          | Variant in exon Y of gene X and reportable                                                                                                                                                                                                                                                                         | `WARN` in case of non-canonical transcript or unreportability                                                                                                                           |
 | MUTATION_IN_GENE_X_IN_EXON_Y_TO_EXON_Z                                                | Variant in codon Y-Z of gene X and reportable                                                                                                                                                                                                                                                                      | `WARN` in case of non-canonical transcript or unreportability                                                                                                                           |
 | MUTATION_IN_GENE_X_IN_EXON_Y_OF_TYPE_Z                                                | Variant in exon Y of type Z (one of: INSERT, INDEL, DELETE, SNV, MNV) in gene X and reportable                                                                                                                                                                                                                     | `WARN` in case of non-canonical transcript or unreportability                                                                                                                           |
-| INACTIVATION_OF_GENE_X                                                                | If geneRole of gene X is not 'ONCO', and (min copy nr < 0.5 or homozygous disruption or biallelic mutation(s) with high driver likelihood, and corresponding event does not match with protein effect gain of function (predicted)                                                                                 | `WARN` in case of gene role ONCO, driver likelihood, other protein effect, biallelic status and/or reportability                                                                        |
+| INACTIVATION_OF_GENE_X                                                                | If geneRole of gene X is not 'ONCO', and (min copy nr < 0.5 or homozygous disruption or biallelic mutation(s) with high driver likelihood, and corresponding event does not match with protein effect gain of function (predicted)                                                                                 | `WARN` in case of gene role ONCO, driver likelihood, other protein effect, biallelic status and/or reportability. Variants estimated to be not clonal are not matched.                  |
```

Comment:
Maybe change the description to "variants estimated to be subclonal are not matched"? Double-not looks odd to me.